src/js/load-pyodide.js

```diff
@@ -43,6 +43,17 @@ export async function initializePackageIndex(indexURL) {
   }
 }
 
+export async function fetchTarFile(indexURL, path) {
```

Comment:
There is nothing specific to .tar here? Maybe `_fetchBinaryFile` ? I would still like to indicate somehow that it is not expected to be used by users...